### PR TITLE
fix!: client with raw abs path option

### PR DESCRIPTION
**Pull request by hacdias**
See https://github.com/ipfs/kubo/pull/10068.

- [x] https://github.com/ipfs/boxo/pull/434
- [x] ⚠️ **Update to Boxo release before merge**\

Will be released in subsequent PR.

### CODE REVIEW

File: writer.go [hunk 1]
```diff
@@ -132,7 +132,7 @@ func (re *writerResponseEmitter) SetLength(length uint64) {
 		return
 	}
 
-	*re.length = length
+	re.length = &length
```
Comment (Jorropo) [inline writer.go:135]: Can you split this in it's own commit in case this SOMEHOW was correct and we want to revert your change ?

Comment (hacdias) [reply writer.go:135]: (It was literally panicking)

Comment (hacdias) [reply writer.go:135]: (But sure)
